client/src/index.jsx

```diff
@@ -79,7 +81,11 @@ class App extends React.Component {
           user: userData.data[0]
         })
       }).then(() => {
-        this.getSleepData();
+        axios.post('/api/getCalories', {user: this.state.user.id}).then((cal) => {
+          this.setState({totalCalories: JSON.parse(cal.data)})
+          this.getSleepData();
```

Comment:
is this correct? (I accidentally deleted it while merging and put it back in there...)

Reply:
solved!